lib/spree/models/line_item_decorator.rb

```diff
@@ -0,0 +1,11 @@
+module Spree
+  module LineItemDecorator
+    protected
+
+    def unit_price
+      price
+    end
```

Comment:
I'd like to avoid decorating models unnecessarily when we could use configuration extension points.

Rather than add a method to the class that others might need to override, we could so something similar to other configurable functionality we already have; we could add a `line_item_unit_price_calculator` option to [the configuration](https://github.com/SuperGoodSoft/solidus_taxjar/blob/master/lib/super_good/solidus_taxjar.rb#L14-L53):

```ruby
attr_accessor :line_item_unit_price_calculator

# ...

self.line_item_unit_price_calculator = ->(spree_line_item) { spree_line_item.price }
```

Reply:
I'm all for that